### PR TITLE
Check in Cloud Job Discovery quick start

### PR DESCRIPTION
**Pull request by xinyunh0929**
CJD got Public beta launch so we would like to have our samples checked in as other projects.

### CODE REVIEW

**Comment by xinyunh0929** [comment]
Hi reviewers,

The test failed in 'gcloud config set project $GOOGLE_CLOUD_PROJECT'
But I don't think we need it in our server. 
Should I still need to set it?
If so, how?

File: jobs/cjd_sample/Gemfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2017 Google, Inc
```
Comment (frankyn) [inline jobs/cjd_sample/Gemfile]: 2018*

File: jobs/cjd_sample/Gemfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source "https://rubygems.org"
+
+gem "google-api-client", "~> 0.23.2"
```
Comment (frankyn) [inline jobs/cjd_sample/Gemfile]: how about `"~> 0.23"` instead for patch versions.

File: jobs/cjd_sample/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google
```
Comment (frankyn) [inline jobs/cjd_sample/README.md]: I recommend borrowing a bit from this [README.md](https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/master/vision/README.md).

File: jobs/cjd_sample/quickstart.rb [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START quickstart]
+require "google/apis/jobs_v2"
+
+# Instantiate the client
+Jobs   = Google::Apis::JobsV2 # Alias the module
```
Comment (frankyn) [inline jobs/cjd_sample/quickstart.rb]: align `=` assignments

File: jobs/cjd_sample/quickstart.rb [hunk 5]
```diff
@@ -0,0 +1,43 @@
+# Copyright 2017 Google, Inc
```
Comment (frankyn) [inline jobs/cjd_sample/quickstart.rb]: 2018

File: jobs/cjd_sample/spec/quickstart_spec.rb [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "rspec"
+require "rspec/retry"
+require "google/apis/jobs_v2"
```
Comment (frankyn) [inline jobs/cjd_sample/spec/quickstart_spec.rb]: this shouldn't be required.

Comment (xinyunh0929) [reply jobs/cjd_sample/spec/quickstart_spec.rb]: It seems I could not run the test via "bundle exec rspec" without these 3 lines.

**Comment by xinyunh0929** [comment]
Done.

File: jobs/cjd_sample/spec/quickstart_spec.rb (hunk 6 above)
Comment (frankyn) [reply jobs/cjd_sample/spec/quickstart_spec.rb]: What error are you seeing?
Reason why I recommend not including it is your quickstart includes this requirement.

**Comment by frankyn** [comment]
Sorry I didn't notice: move files in`cjd_sample` into `jobs/`.

**Comment by xinyunh0929** [comment]
Done.

**Comment by frankyn** [comment]
Pending kokoro CI tests.

**Comment by xinyunh0929** [comment]
Hi Frank,

The test failed in 'gcloud config set project $GOOGLE_CLOUD_PROJECT'
But I don't think we need it in our server.

**Pull request merged**